pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -153,9 +153,13 @@ type ClusterSpec struct {
 	// Optional: ImagePullSecret references a secret with container registry credentials. This is passed to the machine-controller which sets the registry credentials on node level.
 	ImagePullSecret *corev1.SecretReference `json:"imagePullSecret,omitempty"`
 
+	// Optional: CNIPlugin refers to the spec of the CNI plugin used by the Cluster.
 	CNIPlugin *CNIPluginSettings `json:"cniPlugin,omitempty"`
 
-	ClusterNetwork  ClusterNetworkingConfig   `json:"clusterNetwork"`
+	// Optional: ClusterNetwork specifies the different networking parameters for a cluster.
+	ClusterNetwork ClusterNetworkingConfig `json:"clusterNetwork"`
+
+	// Optional: MachineNetworks refers to array of the networking parameters used for IPAM.
```

Comment:
I think in general the expression "refers to" is misleading here and in all other spots you used it in this PR. This field here, MachineNetworks, does not _refer to_ an array, it _is an array_. A better comment woul be

> Optional: MachineNetworks is a list of the networking parameters used for IPAM.